sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -2,7 +2,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.protobuf.TextFormat;
-import com.mesosphere.sdk.scheduler.recovery.RecoveryPlanManagerFactory;
+import com.mesosphere.sdk.scheduler.recovery.*;
```

Comment:
I think there's some lint check that complains if too many classes from a package are imported separately. I don't remember what the limit is, but that could be why this was switched to a wildcard?

While in here it could make sense to just clean up the import ordering since it looks pretty haphazard at this point.